usr/share/regolith-look/ayu-dark/i3-wm

```diff
@@ -1,8 +1,10 @@
-#define i3wm_window_border_size         3
-#define i3wm_floatingwindow_border_size 3
-#define i3wm_gaps_inner_size            5
-#define i3wm_gaps_outer_size            0
-#define i3wm_bar_position               bottom
+#define i3wm_window_border_style         pixel
```

Comment:
Yes, it will ensure that if a user switches to the 'i3-default' look and then back to the 'default' look that the titlebar styling is set correctly.

I forgot about the default look in all honesty.. I will raise a PR shortly to add these.

Reply:
I have added the PR for the default look here:
https://github.com/regolith-linux/regolith-look-default/pull/4

Reply:
Thanks!